test/main.go

```diff
@@ -1,70 +0,0 @@
-package main
```

Comment:
We should keep this. This is a helper binary to check if reuseport is working on the current machine (helps with debugging).

Reply:
I think this is well covered by the unit tests. The example binary was confusing and long-winded.

Reply:
IIRC, this binary exists for making sure that reuseport works on the local machine (we've had a *lot* of problems with reuseport in the past). We can ship this off to a user to rule out reuseport issues.

Reply:
We can ship the unit test binary, `go test -c`, which will be much more comprehensive and appropriate. I've used this to test with wine.

Reply:
As long as it emits enough information for us to debug.